composer/trainer/devices/device_hparams_registry.py

```diff
@@ -18,4 +18,5 @@
 device_registry: Dict[str, Union[Type[Device], Type[hp.Hparams]]] = {
     'gpu': DeviceGPU,
     'cpu': DeviceCPU,
+    'amd_gpu': DeviceGPU,
```

Comment:
Is this necessary? If we're mapping directly to `DeviceGPU` it seems to imply we might be able to just use 'gpu' for both NVIDIA and AMD :D

Reply:
right, but i want to still have this option to clearly distinguish runs.

Reply:
Agree with @abhi-mosaic; this is a bit confusing / redundant...

Curious what is the use case to distinguish? is it for logging? If so, then can we have the trainer add a log statement for the cuda device id or the like?

Reply:
+1 on removing this and just using the `gpu` device if possible, we can either log or find another way to capture the device specific information.

How does this work?  The `DeviceGPU` class defined in the `device_gpu.py` module is a CUDA specific implementation.  My understanding is that AMD uses a different backend called ROCm so wouldn't this need to change?